WcaOnRails/app/webpacker/components/Panel/Board/LeaderChangeForm.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import { Form } from 'semantic-ui-react';
+import WcaSearchFormControl from '../../SearchWidget/WcaSearchFormControl';
+import useSaveAction from '../../../lib/hooks/useSaveAction';
+import Loading from '../../Requests/Loading';
+import { COUNCILS_STATUS } from '../../../lib/helpers/groups-and-roles-constants';
+import { apiV0Urls } from '../../../lib/requests/routes.js.erb';
+
+const oldLeaderStatusOptions = ['senior_member', 'member', 'resign'];
+const oldLeaderStatusOptionsMap = {
+  senior_member: 'Senior Member',
+  member: 'Member',
+  resign: 'Resign',
+};
+
+export default function LeaderChangeForm({
+  setEditLeader, syncData, group, oldLeader,
+}) {
+  const [formValues, setFormValues] = useState({
+    newLeader: null,
+    oldLeaderStatus: oldLeaderStatusOptions[0],
+  });
+  const { save } = useSaveAction();
+  const [saving, setSaving] = useState(false);
```

Comment:
The `useSaveAction` hook already provides `saving`, I don't think we need this state. There's the sync and the state reset after the save calls, but neither of those is `await`ed so I don't think it matters.

Reply:
Thing is, I have two actions to be made and one should happen after the other request is *completed*. If I use the `saving` of `useSaveAction`, it will have a very short time in between the requests where the `saving` will be false right?